xbmc/music/MusicInfoLoader.cpp

```diff
@@ -102,19 +101,21 @@ bool CMusicInfoLoader::LoadAdditionalTagInfo(CFileItem* pItem)
     if (database.GetAlbum(param.GetAlbumId(), album, false))
       CMusicDatabase::SetPropertiesFromAlbum(*pItem,album);
 
-    path = pItem->GetMusicInfoTag()->GetURL();
+    // since we're going to load additional tag information we
+    // need to replace the library path with the actual file path.
+    // this has to be done before creating the infotag loader.
+    pItem->SetPath(pItem->GetMusicInfoTag()->GetURL());
```

Comment:
> Do we need to set the path in the CFileItem so that the call to CMusicInfoTagLoaderFactory::CreateLoader creates the proper loader?

I've changed it to load by item in 3c1145b9a40c88f66c19ab16bf8b74898777b376 which is causing the lyrics issue. What about creating a dummy CFileItem based on the translated path?